src/containers/Trade.svelte

```diff
@@ -95,15 +95,11 @@
     : TradeButtonStatus.Trade;
 
   // reset amounts and best order on network change
-  detectProvider('marina')
-    .then((marina: MarinaProvider) => {
-      marina.on('NETWORK', () => {
-        bestOrder = undefined;
-        receiveAmount = undefined;
-        sendAmount = undefined;
-      });
-    })
-    .catch(console.error);
+  marinaStore.subscribe(() => {
```

Comment:
this will be triggered everytime connected, installed or network changes. Is that what you wanted?

If yes, let's leave a comment

Reply:
Yes, it is what I wanted.

Commit d4d0a2ca fixes this.